site/index.html

```diff
@@ -9,42 +9,30 @@ <h1>Join Us</h1>
 
 <p>freeCodeCamp Oklahoma is a group for those who want to learn to code throughout the Oklahoma state area.</p>
 
-<h2 id="meetings">Meetings</h2>
+<h2 id="meetings">Meetings</h2
+<p class="meet-element">
```

Comment:
We don't need this class anymore so we can take it out. 

You can also search the project for `meet-element` and `meet-wrapper` and remove those unused CSS styles if you're feeling up for it.

Reply:
I removed the 'meet-element' class from index.html, and both the 'meet-element' and 'meet-wrapper' styles from styles.css. In the later, I actually removed the whole "meet wrapper" section of CSS code, as it appeared to all be unused now.